src/applications/claims-status/components/claim-document-request-pages/DefaultPage.jsx

```diff
@@ -88,6 +93,30 @@ export default function DefaultPage({
           </>
         )}
       </h1>
+      {/* Only show errors in DefaultPage when feature flag is ON */}
+      {/* For type 1 known errors, display an alert */}
+      {message && (
+        <div className="vads-u-margin-top--0">
+          <Notification
+            title={message.title}
+            body={message.body}
+            type={message.type}
+            onSetFocus={focusNotificationAlert}
+          />
+        </div>
+      )}
+      {/* For type 1 unknown errors, display the Type 1 Unknown Upload Error alert */}
+      {type1UnknownErrors &&
+        type1UnknownErrors.length > 0 && (
+          <div className="vads-u-margin-y--4">
+            <Notification
+              title="We need you to submit files by mail or in person"
+              body={<Type1UnknownUploadError errorFiles={type1UnknownErrors} />}
+              type="error"
+              onSetFocus={!message ? focusNotificationAlert : undefined}
```

Comment:
Updated this to only focus if there's no known error alert present, as we want the priority to go to known errors.